src/Countries/US/Oregon/WorkersCompAssessmentFundEmployer/V20200101/WorkersCompAssessmentFundEmployer.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\Oregon\WorkersCompAssessmentFundEmployer\V20200101;
+
+use Appleton\Taxes\Countries\US\Oregon\WorkersCompAssessmentFundEmployer\WorkersCompAssessmentFundEmployer as BaseWorkersCompAssessmentFundEmployer;
+use Illuminate\Database\Eloquent\Collection;
+
+class WorkersCompAssessmentFundEmployer extends BaseWorkersCompAssessmentFundEmployer
+{
+    const HOURLY_RATE_MODIFIER = 1.1;
+
+    public function compute(Collection $tax_areas)
+    {
+        $this->tax_total = $this->payroll->withholdTax($this->payroll->getHoursWorked($tax_areas->first()->workGovernmentalUnitArea) * static::HOURLY_RATE_MODIFIER);
```

Comment:
looks like we might have issues with the wages `minutes_worked` for salaries and sick leave. Not related to this PR I don't think but would hit this tax.

Reply:
Hmm, ya. So salaries and sick leave don't have minutes worked?

Reply:
looks like they have the wrong work minutes. sick leave is using pay rate for example.

Reply:
So how do we solve this? I am not coming up with anything off the top of my head